### DISCUSSION TITLE
how to capture my keyboard input <c-l> when we open terminal by <c-/> ?

### DISCUSSION THREAD

**Discussion by Fodesu** [LazyVim/LazyVim#2809] [Q&A]
i want to use `<c-l> `to clear terminal by <c-l> will capture by  `Go to right window`

**Comment by dpetka2001** [comment]
Just write `clear` in the terminal or disable the keymap `<C-l>`, which by default in LazyVim is used for moving across Neovim windows.

**Reply by Fodesu** [reply]
i know `clear` can clear terminal, but i perfer `<C-l>` 
Can I configure it to only disable <C-l> when the terminal is open?

**Reply by dpetka2001** [reply]
Put in your `/lua/config/keymaps.lua` the following `vim.keymap.del("t", "<C-l>")`. Also please make sure to read the LazyVim docs. This is mentioned in the [docs](https://www.lazyvim.org/configuration/keymaps).
